src/components/Sing/ScoreSequencer.vue

```diff
@@ -53,6 +53,17 @@
               :height="gridCellHeight"
               :class="`sequencer-grid-cell sequencer-grid-cell-${keyInfo.color}`"
             />
+            <!-- オクターブのライン / オクターブのみに引きたい -->
+            <line
+              v-for="(keyInfo, index) in keyInfos"
+              :key="index"
+              x1="0"
+              :x2="gridCellWidth"
+              :y1="gridCellHeight * index"
+              :y2="gridCellHeight * index"
+              stroke-width="1"
+              :class="`sequencer-grid-p-line-${keyInfo.pitch.toLowerCase()}`"
+            />
```

Comment:
たぶんこんな感じで行けるかと！！
（`.sequencer-grid-pitch-line-c`を`.sequencer-grid-pitch-line`にしつつ）
```suggestion
            <template v-for="(keyInfo, index) in keyInfos" :key="index">
              <line
                x1="0"
                :x2="gridCellWidth"
                :y1="gridCellHeight * index"
                :y2="gridCellHeight * index"
                stroke-width="1"
                class="sequencer-grid-p-line"
              />
            </template>
```

Reply:
ああ。。。すみません、これifが漏れてました。。こうできるかもです！ 🙇 
（今のoctaveKeyIndexesでももちろん 🙆‍♀️ ）

```vue
            <template v-for="(keyInfo, index) in keyInfos" :key="index">
              <line
                v-if="keyInfo.pitch === 'C'"
                x1="0"
                :x2="gridCellWidth"
                :y1="gridCellHeight * index"
                :y2="gridCellHeight * index"
                stroke-width="1"
                class="sequencer-grid-p-line"
              />
            </template>
```

Reply:
@Hiroshiba 
ありがとうございます！octaveKeyIndexes持つ必要ないので、こちらの方がよさそうです！
こちら修正いたしました！